packages/node-cli/src/SDKExecutor.js

```diff
@@ -39,7 +40,7 @@ module.exports.SDKExecutor = class SDKExecutor {
 			? this._authenticationService.getProjectDefaultAuthId()
 			: null;
 
-		return new Promise((resolve, reject) => {
+		return new Promise(async (resolve, reject) => {
```

Comment:
What does async solve here?

Reply:
We can only use await inside an async method, and since we need to await SDKDownloadService.download() in line 56 we need to transform this into an async method